superset/assets/src/dashboard/components/ColorComponentPane.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-env browser */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { getCategoricalSchemeRegistry } from '@superset-ui/color';
+import { t } from '@superset-ui/translation';
+
+import ColorSchemeControl from '../../explore/components/controls/ColorSchemeControl';
+import { BUILDER_PANE_TYPE } from '../util/constants';
+
+const propTypes = {
+  showBuilderPane: PropTypes.func.isRequired,
+  setColorSchemeAndUnsavedChanges: PropTypes.func.isRequired,
+  colorScheme: PropTypes.string,
+};
+
+const defaultProps = {
+  colorScheme: undefined,
+};
+
+class ColorComponentPane extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { hovered: false };
+    this.categoricalSchemeRegistry = getCategoricalSchemeRegistry();
+    this.getChoices = this.getChoices.bind(this);
+    this.getSchemes = this.getSchemes.bind(this);
+    this.onCloseButtonClick = this.onCloseButtonClick.bind(this);
+    this.onMouseEnter = this.setHover.bind(this, true);
+    this.onMouseLeave = this.setHover.bind(this, false);
+  }
+
+  onCloseButtonClick() {
+    this.props.showBuilderPane(BUILDER_PANE_TYPE.NONE);
+  }
+
+  getChoices() {
+    return this.categoricalSchemeRegistry.keys().map(s => [s, s]);
+  }
+
+  getSchemes() {
+    return this.categoricalSchemeRegistry.getMap();
+  }
+
+  setHover(hovered) {
+    this.setState({ hovered });
+  }
+
+  render() {
+    const { setColorSchemeAndUnsavedChanges, colorScheme } = this.props;
+
+    return (
+      <div className="slider-container">
+        <div className="component-layer slide-content">
+          <div className="dashboard-builder-sidepane-header">
+            <span>{'Color Settings'}</span>
+            <i
+              className="fa fa-times trigger"
+              onClick={this.onCloseButtonClick}
+              role="none"
+            />
+          </div>
+          <div
+            className="panel-body"
+            onMouseEnter={this.onMouseEnter}
+            onMouseLeave={this.onMouseLeave}
+          >
+            <ColorSchemeControl
```

Comment:
can you add scheme name as tooltip when user mouse over color scheme choices?

Reply:
Let me get @dorq 's opinion. The color name schemes may not make sense to users based on what we call them today.

Reply:
@khtruong and I discussed this offline, and we agreed that even though the color scheme names are not very user-friendly (e.g. 'lyftColors', 'bnbColors', 'd3Category10', d3Category20'), there are situations in which it would be valuable to see them.  So adding them as a tooltip (as opposed to putting the scheme names directly into the dropdown options) is a good compromise.

@khtruong will add this to the PR (thanks, Kim!)

Reply:
I'll try to make them as user-friendly as possible (i.e. lyftColors -> Lyft Colors, bnbColors -> Airbnb Colors, etc)

Reply:
i don't really mean to use scheme name (like lyftcolors bnbcolors), any user friendly descriptive name will be good. so that when i want to apply same color scheme all over multiple dashboards, it will be easier for me to remember.

Reply:
<img width="409" alt="Screen Shot 2019-03-28 at 10 23 29 AM" src="https://user-images.githubusercontent.com/47833996/55180714-62957280-5147-11e9-95d5-121f1ffd1ab4.png">

<img width="412" alt="Screen Shot 2019-03-28 at 10 23 43 AM" src="https://user-images.githubusercontent.com/47833996/55180718-645f3600-5147-11e9-996e-2319ae201435.png">

I left scheme names as is unless someone has better names.

Reply:
LGTM

Reply:
`ColorScheme` datastructure has `label` field.
I think the code in this PR should use the `label` instead of the `keys` which are `id`.
Then we can update the color schemes in `@superset-ui` to include human-friendly `label`.

p.s. `ColorScheme` also has `description` field.

Reply:
![image](https://user-images.githubusercontent.com/1659771/55181281-9b821700-5148-11e9-900e-9a729c9d71bd.png)

Reply:
I just updated the color schemes in @superset-ui to have actual labels.

Reply:
https://github.com/apache-superset/superset-ui/pull/125

Reply:
awesome!